src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,81 @@
+import cn from 'classnames';
+import { useContext } from 'react';
+import { Status } from '../types/Status';
+import { TodoContext } from './TodoContext';
+
+export const TodoFooter: React.FC = () => {
+  const {
+    todos, filter, setFilter, deleteTodo,
+  } = useContext(TodoContext);
+
+  const activeTodoCount = todos.filter(todo => !todo.completed).length;
+  const activeTodoCountMsg = `${activeTodoCount} items left`;
+
+  const hasCompletedTodos = todos.some(todo => todo.completed);
+
+  const handleClearCompleted = () => {
+    todos.filter(todo => todo.completed).forEach(todo => deleteTodo(todo.id));
+  };
+
+  if (!todos.length) {
+    return <></>;
+  }
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodoCountMsg}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments)